crates/eips/src/eip2930.rs

```diff
@@ -141,6 +141,31 @@ pub struct AccessListWithGasUsed {
     pub gas_used: U256,
 }
 
+
+/// `AccessListResult` for handling errors from `eth_createAccessList`
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct AccessListResult {
```

Comment:
this needs rename to camelcase

Reply:
https://github.com/alloy-rs/alloy/pull/1110/commits/93c58717c10196317ab1d08962e92caea2cf4309